<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoSysSettingsDisplayForm</class>
 <widget class="QWidget" name="GeoSysSettingsDisplayForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Phi Sector Cutaways</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="PhiSectionWidget" name="phisectionwidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Draw options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Transparency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_transp">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_showVolumeOutLines">
        <property name="toolTip">
         <string>Decorate shapes with lines at the edges (current only certain shapes such as muon chambers are supported)</string>
        </property>
        <property name="text">
         <string>Show outlines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="VP1DrawOptionsWidget" name="widget_drawOptions" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_colouring">
     <property name="title">
      <string>Custom Colouring</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="_8">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Edit material of last selected volume:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VP1MaterialButton" name="matButton_lastSel">
          <property name="toolTip">
           <string>Edit the material of the last picked volume</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_saveChangedMaterials">
        <property name="toolTip">
         <string>Store changed materials into a .vp1geomat file which can be loaded later (hold down CTRL while clicking to store ALL materials, rather than just the ones that do not have default values)</string>
        </property>
        <property name="text">
         <string>Save changed materials</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_loadMaterials">
        <property name="toolTip">
         <string>Load materials from a .vp1geomat file</string>
        </property>
        <property name="text">
         <string>Load materials</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_4">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1MaterialButton</class>
   <extends>QPushButton</extends>
   <header>VP1Base/VP1MaterialButton.h</header>
  </customwidget>
  <customwidget>
   <class>VP1DrawOptionsWidget</class>
   <extends>QWidget</extends>
   <header>VP1Base/VP1DrawOptionsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhiSectionWidget</class>
   <extends>QGraphicsView</extends>
   <header>VP1Base/PhiSectionWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
